MeshLib/MeshQuality/MeshValidation.h

```diff
@@ -65,7 +66,26 @@ class MeshValidation
 	static std::array<std::string, static_cast<std::size_t>(ElementErrorFlag::MaxValue)>
 		ElementErrorCodeOutput(const std::vector<ElementErrorCode> &error_codes);
 
+	/** 
+	 * Tests if holes are located within the mesh. 
+	 * In this context, a hole is a boundary of an element with no neighbor that cannot be reached from
+	 * the actual boundary of the mesh. Examples include a missing triangle in a 2D mesh or a missing 
+	 * Tetrahedron in a 3D mesh.
+	 * Note, that this method does not work when complex 3D structures are build from 2D mesh elements,
```

Comment:
mmmh ... I think this note makes sense here, as *this* method will give completely wrong results in the given case.
I will add an additional note to ```getMeshBoundary()``` because the result might not be exactly what the user is expecting. (It is debatable wether the result is correct or not in this case, depending on the expectations you have when you call the method).